templates/main_template.yml

```diff
@@ -9,6 +9,8 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
+    permissions:
+      contents: write
```

Comment:
It is now required by Github Actions to have this permission to make releases. If the action doesn't have it, the last step will fail.